snap/snapcraft.yaml

```diff
@@ -835,3 +858,100 @@ parts:
         -e "s:host = \"edgex-support-rulesengine\":host = \"localhost\":" \
         -e "s:host = \"edgex-device-virtual\":host = \"localhost\":" \
         $SNAPCRAFT_PART_INSTALL/config/security-api-gateway/res/configuration.toml
+  device-modbus:
+    source: https://github.com/edgexfoundry/device-modbus-go.git
+    source-depth: 1
+    #source-branch: dehli
```

Comment:
I see that this, `device-mqtt-go` and `device-random` all only have a single master branch and no tags. As such I would rather see all these pin to specific to the SHA's  that those repos' master is currently on, otherwise as daily builds continue after the stable release more recent code will automatically for these services will be pulled into `delhi/edge` releases, and if those repos ever make breaking changes `delhi/edge` will become automatically broken at some point...

Reply:
Actually nevermind, we talked about this and we don't want to tag just yet